src/twisted/internet/task.py

```diff
@@ -837,7 +837,7 @@ def pump(self, timings):
 
 
 
-def deferLater(clock, delay, callable, *args, **kw):
+def deferLater(clock, delay, callable=None, *args, **kw):
```

Comment:
It would be nice if the docs covered this in some cursory sense, but it does seem reasonably obvious what will happen if you don't pass it, so I'm OK with that being a follow-up if you like.